src/whatsnew.adoc

```diff
@@ -95,95 +102,91 @@ their position in the table (i.e., the `$id` syntax).
 
 In Ion 1.0, all data is length prefixed.  While this is good for optimizing the reading of data, it requires an Ion
 encoder to buffer any data in memory to calculate the data's length.  Ion 1.1 introduces optional binary syntax to allow
-containers to be encoded with an end marker.
-
-==== Low-level Encoding Changes
-
-The `VarUInt` and `VarInt` encodings have been changed to put their continuation bits before the content.  This is
-similar in structure to UTF-8 encoding.  `VarInt` has also been changed to be a two's complement representation rather
-than a signed magnitude representation.  Benchmarks have shown that these encoding changes to be about 15% faster than
-the Ion 1.0 version of the encoding.
-
-The `Int` encoding has also been changed to be two's complement over its original signed magnitude representation.
-
-IMPORTANT: *_Discussion_*: Should we call these new low-level encodings something else to not confuse them with their
-1.0 counterparts?  If so, what should we call them?
-
-A new primitive encoding type, called `VarSym` has been introduced to flexibly encode symbol IDs and symbolic tokens
-with inline text.  This encoding is structurally identical to `VarInt` but is used to encode symbolic tokens that could
-be either a symbol ID or a length of the UTF-8 octets that follow.
-
-* Positive integers represent symbol IDs.
-
-* Negative integers represent a length of UTF-8 octets to follow.
-
-* Zero is treated specially.  Specifically, A `VarSym` encoded as `0x00` indicates that a special control opcode
-follows. The following table specifies the control opcodes, all other opcodes are illegal.
-
-[%header,%unbreakable,cols="1,1"]
-|===
-
-| Control Opcode
-| Description
-
-| `0x0_` - `0x4_`, `0xE1`, `0xF0`
-| E-expression follows.  This is only valid when a field name in a struct is expected.
-
-| `0x70`
-| Represents `$0`.
-
-| `0x80`
-| Represents empty inline text (i.e., `''`).
-
-| `0xAD`
-| Ends a delimited struct.  This is only valid when a field name in a _delimited_ struct is expected.
-
-|===
-
-The padded representations of zero in `VarSym` (e.g., `0x80 0x00`, `0xC0 0x00 0x00`) all mean `$0`.
-
-NOTE: The _control opcode_ is designed to parallel the Ion 1.1 _type opcodes_ which is why they might seem, out of that
-context, arbitrary.
+containers to be encoded with an end marker instead of a length prefix.
+
+==== Low-level Binary Encoding Changes
+
+Ion 1.0's link:https://amazon-ion.github.io/ion-docs/docs/binary.html#varuint-and-varint-fields[`VarUInt` and `VarInt`
+ encoding primitives] used big-endian byte order and used the high bit of each byte to indicate whether it was the final
+byte in the encoding.
+`VarInt` used an additional bit in the first byte to represent the integer's sign.
```

Comment:
This feels like an unnecessary level of detail for this overview.  I suggest we lead by saying that 1.1 leans into little-endianness (and why) and thus replaces X and Y with P and Q.

More detailed "delta" narrative would fit better into the intro to the binary chapter.

Reply:
Agreed. Tracking this in #239.